### ISSUE TITLE
error NU1108 Cycle detected 

### ISSUE THREAD

**Issue by dotChris90** [riganti/dotvvm#525]
Hello dotvvm Team, 

I looked your amazing dotvvm framework some weeks ago and did the tutorials. 
Now when europe is in christmas mood and we all find some more time for experimenting ;) 
I want to try out dotvvm on my windows 7 laptop (my idea was to maybe combine dotvvm with ELECTRON.NET / quite new project too). 

Unfortunately .... my motivation was high but I faceing some trouble. When I added the DotVVM.AspNetCore .... and restore ..... it say "error NU1108 Cycle detected" and "dotVVM --> DotVVM.ASpNetCore 1.1.6 --> DotVVM (>= 1.1.6)" 

The csproj file is the following : 

<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp2.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <Folder Include="wwwroot\" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="DotVVM.AspNetCore" Version="1.1.6" />
    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
  </ItemGroup>

</Project>

I already tried out go down with version but (if i remember your blogs) asp.net core 2.0 is just supported by 1.1.6 - so i think going down makes so sence. 

I am not very sure what could be the problem since the error message .... is not very self explaining xD (to me) or not very detailed. 

I saw that you already got such a ticket in Mai 2017 "Cycle detected: dotvvm (>= 1.0.0) -> DotVVM.AspNetCore (>= 1.1.0.1) -> DotVVM (>= 1.1.0.1). #361". I just can say "downgrade solution" does not work xD Any other people got this issue? 

Maybe later I can tell you if it is the same on Linux machines or just windows. But for now just can say : windows 7 x64. 

Thanks for your time and merry christmas :)
dotChris90 / Christian

**Comment by tomasherceg** [comment]
I have seen this error message several times but don't know the reason - maybe some Nuget issue.

I would try these steps, one of them should help:
* downloading the latest .NET Core 2.0 SDK
* make sure that you are referencing Microsoft.AspNetCore 2.0, not 1.x
* upgrade all DotVVM packages to 1.1.7
* clean, rebuild, delete bin and obj folders and try again

**Comment by exyi** [comment]
What it the name of your project? I'd guess from the error message that it is `dotvvm` so `dotnet` thinks that `DotVVM.AspNetCore` references your project. I have just tried and results in exactly the same error message (note that it's case insensitive, even on Linux)

**Comment by dotChris90** [comment]
@exyi  @tomasherceg  thanks for your reply - this was the fastest solution I got ever. 

@exyi I have to admit .... yes .... most stupid mistake ever. -.- I just created a project with title "HelloDotVVM" - added dotvvm 1.1.7 - and ..... it worked. Oh god - feel little bit ashamed. xD 

Haha much thanks both of you. I did not expect to go on so fast ;) 

And wish you a happy christmas time. :)
 Greeting 
dotChris90 / Christian
